.github/workflows/integration-tests.yaml

```diff
@@ -55,6 +55,22 @@ jobs:
             python-version: "3.10"
 
     steps:
+      # This step is a Workaround to avoid the "No space left on device" error.
+      # ref: https://github.com/actions/runner-images/issues/2840
+      - name: Remove unnecessary files
+        shell: bash
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+          sudo rm -rf /usr/local/lib/android
+          sudo rm -rf /usr/local/share/powershell
+          sudo rm -rf /usr/share/swift
+          
+          echo "Disk usage after cleanup:"
+          df -h
```

Comment:
To avoid `No space left on device` error:

https://github.com/kubeflow/training-operator/actions/runs/6338749256